src/Altinn.Common.PEP/Altinn.Common.PEP/Helpers/DecisionHelper.cs

```diff
@@ -27,6 +27,7 @@ public static class DecisionHelper
         private const string ParamApp = "app";
         private const string ParamOrg = "org";
         private const string ParamAppId = "appId";
+        private const string ParamWho = "who";
```

Comment:
Not a big fan of the "who" parameter. Actor has been mentioned as a 
possibility

Reply:
Fixed